src/olympia/reviewers/serializers.py

```diff
@@ -213,6 +216,20 @@ def get_content(self, obj):
         # more explanation.
         return ''
 
+    def get_uses_unknown_minified_code(self, obj):
+        try:
+            validation = obj.validation
+        except FileValidation.DoesNotExist:
+            # We don't have any idea about whether it could be minified or not
+            # so let's assume it's not for now.
+            return False
+
+        validation_data = json.loads(validation.validation)
+
+        prop = 'unknownMinifiedFiles'
```

Comment:
Can't this be inlined instead of having a variable used one time?

Reply:
I found it much easier to read that way. In-lining it would mean breaking the following statement through multiple lines and that will make it much harder to read imo.